sli/interpret.h

```diff
@@ -898,6 +905,18 @@ SLIInterpreter::addmodule( void )
 }
 
 inline void
+SLIInterpreter::reset_modules()
```

Comment:
Why should this function be inlined? It is in no way performance critical and not a template, so it could very well go into the `cc` file.